content/content.js

```diff
@@ -13,27 +13,16 @@ const submitUrl = 'https://borang.skrin.xyz/submit';
   document.documentElement.append(div);
 })();
 
-let submitButton = document.querySelector(
-  // '.freebirdFormviewerViewNavigationSubmitButton'
-  // '.uArJ5e.UQuaGc.Y5sE8d.VkkpIf.NqnGTe'
-  '.uArJ5e.UQuaGc.Y5sE8d.VkkpIf'
-);
+const submitButton = document.querySelector('.uArJ5e.UQuaGc.Y5sE8d.VkkpIf.QvWxOd');
 
 if (submitButton) {
-  injectScript();
-}
-
-function injectScript() {
-  let script = document.querySelector('body > script');
-  let clone = script.cloneNode(true);
-  clone.setAttribute('type', 'text/javascript');
-  let body = document.querySelector('body');
-  body.appendChild(clone);
   const form = document.querySelector('form');
-  const formUrl = form.action;
-  form.method = 'POST';
-  form.action = submitUrl;
-  submitButton.onclick = async (e) => {
+
+  submitButton.onclick = () => {
+    const formUrl = form.action;
```

Comment:
This part was the fix where form is still submitted after user clicked `Back` button on form last page